src/engine/core/blob.h

```diff
@@ -23,8 +23,7 @@ namespace Lumix
 			int getPos() const { return m_pos; }
 			void write(const void* data, int size);
 			void writeString(const char* string);
-			template <class T> void write(const T& value) { write(&value, sizeof(T)); }
```

Comment:
I found a way to specialize `read` and `write` without overloading (because for `read` case you should provide explicit type parameter and overloading is not sufficient).